test/WebApplication1/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
```

Comment:
Can we rename current sample app that we have to Mvc6Framework45TestApp and this one to WebApiShimFw46TestApp or similar to reflect what is what? (I suppose you used a different template, different fw version and you will uncomment web api shim so we can test request names)